charts/zipkin/ci/elasticsearch-values.yaml

```diff
@@ -0,0 +1,13 @@
+---
+zipkin:
+  storage:
+    type: elasticsearch
+    elasticsearch:
+      hosts: http://localhost:9200
+
+# extra containers are in the same pod, so accessible by localhost
+extraContainers:
```

Comment:
reason this works without any other machinery is that zipkin checks its dependencies and doesn't return OK healthcheck until they are ok.

Hence, it can resolve this as you'll see in the logs.. then the wget tests are safe to run. pretty handy

```
   Warning  Unhealthy  20s                kubelet            Readiness probe failed: Get "http://10.244.0.7:9411/health": dial tcp 10.244.0.7:9411: connect: connection refused
  Warning  Unhealthy  18s                kubelet            Readiness probe failed: Get "http://10.244.0.7:9411/health": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
  Warning  Unhealthy  11s (x2 over 16s)  kubelet            Readiness probe failed: HTTP probe failed with statuscode: 503
```